cime_config/cesm/allactive/config_compsets.xml

```diff
@@ -46,37 +46,42 @@
   </compset>
 
   <compset>
-    <alias>B1850RG</alias>
+    <alias>B1850RGWs</alias>
```

Comment:
I think we need only 1 B1850 compset alias at this point and that is you B1850 compset. We do not need the other aliases.
Similarly I think we need only 1 BHIST that corresponds to the B1850.  I think we only need to test these.
From the point of view of the fully active compsets I would err on a very sparse set and let any deviation be a user defined compset. Before you implement these changes lets sit down and talk to Jim as well.